WPMedia/fusion-news-theme-blocks
# TMEDIA-730 - Article body block

## Description

Convert Article Body Block to v2 (Enhanced Styling) architecture

_Line Height is not addressed in this PR due to changes forth coming with line heights across the design system_

## Jira Ticket

- [TMEDIA-730](https://arcpublishing.atlassian.net/browse/TMEDIA-730)

## Test Steps

Ensure you .env file is pointing at Themes Internal APIs

1. Checkout this branch `git checkout TMEDIA-730-article-body-block`
2. Checkout out the Feature Pack PR `git checkout TMEDIA-730-article-body-block`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block`
4. In PageBuilder
    * Create a template - "Article - Right Rail"
    * Use Right Rail Layout
    * Add the chain to the Main section of the layout - Article Body Block
    * Update Preview content to use `content-api` and `website_url` to be `/2022/06/08/kitchen-sink/`

## Dependencies or Side Effects

**NOTE** - Secondary font has been set to `Georgia` and has an impact on other stories in Chromatic, you'll see very faint changes highlighted in the diff's which is acceptable due to the font family change

Feature Pack PR - https://github.com/WPMedia/arc-themes-feature-pack/pull/274

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
